src/Uno.UWP/System.Runtime/WindowsRuntimeBufferExtensions.cs

```diff
@@ -1,25 +1,40 @@
-﻿
-
-using System.IO;
+﻿using System.IO;
 using Windows.Storage.Streams;
-using Windows.Foundation;
+using System.Linq;
+using UwpBuffer = Windows.Storage.Streams.Buffer;
 
 namespace System.Runtime.InteropServices.WindowsRuntime
 {
 	public static class WindowsRuntimeBufferExtensions
 	{
-		[Uno.NotImplemented]
-		public static IBuffer AsBuffer(this byte[] source){ throw new NotImplementedException(); }
-		[Uno.NotImplemented]
-		public static IBuffer AsBuffer(this byte[] source, int offset, int length){ throw new NotImplementedException(); }
-		[Uno.NotImplemented]
-		public static IBuffer AsBuffer(this byte[] source, int offset, int length, int capacity){ throw new NotImplementedException(); }
+		public static IBuffer AsBuffer(this byte[] source) =>
+			AsBuffer(source, 0, source.Length);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'source' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)